docs/src/components/PropTable.js

```diff
@@ -72,7 +71,7 @@ export default function PropTable({ props: properties, Component }: Props) {
     if (missingProps.length > 0) {
       // eslint-disable-next-line no-console
       console.warn(
-        `${displayName} is missing ${
+        `${displayName || ''} is missing ${
```

Comment:
@pfarejowicz should we set the default in the destructuring higher up?

`const { displayName = '', propTypes } = Component;`

Reply:
I tried this first too but `displayName` is of type `?string` inside of `React.ComponentType` which means doing `displayName = '',` will catch the `undefined` case but not the `null` case. Flow would still error if we did that so we need to `|| ''` to catch both cases